test/svm/SvmSpoke.Bundle.ts

```diff
@@ -643,167 +656,215 @@ describe("svm_spoke.bundle", () => {
     }
   });
 
-  it("Execute Max Refunds", async () => {
-    const relayerRefundLeaves: RelayerRefundLeafType[] = [];
+  describe("Execute Max Refunds", () => {
+    const executeMaxRefunds = async (refundType: RefundType) => {
+      const relayerRefundLeaves: RelayerRefundLeafType[] = [];
+      // Higher refund count hits inner instruction size limit when doing `emit_cpi` on public devnet. On localnet this is
```

Comment:
to test this on testnet did you use the [script](https://github.com/across-protocol/contracts/blob/master/scripts/svm/simpleFakeRelayerRepayment.ts)?

Reply:
it would be wise I think for us to run this same test on mainnet to ensure that there are no other memory issues that we don't know about on this. does mean deploying the whole program, but that's ok as we can always close it afterwards.

Reply:
Yes, [here](https://explorer.solana.com/tx/2Smpe9DHWfLxS49gbnbEjeoGYhSYeRMfUcmDLyqwt6z6GVcGza1daJ7NHxKpfKKe5pU5cp7rJqXybXfm6nA3qsgb?cluster=devnet) is the refund execution for 28 relayers. At 29 count would get this error: `Instruction passed to inner instruction is too large (1306 > 1280)` when doing `emit_cpi`.

Though, we might need to improve the script as in some runs it closed the instruction parameter account before leaf execution tx got mined.